test/bin/pin-depends.t/run.t

```diff
@@ -0,0 +1,66 @@
+We have a simple project with a single package defined at the root.
+It has a `x-opam-monorepo-opam-repositories` field set to use a local
+opam-repository for locking and a pinned package.
+
+  $ cat pin-depends.opam
+  opam-version: "2.0"
+  depends: [
+    "dune"
+    "b"
+    "c"
+  ]
+  pin-depends: ["b.dev" "./duniverse/b"]
+  x-opam-monorepo-opam-provided: ["b"]
+  x-opam-monorepo-opam-repositories: [
+    "file://$OPAM_MONOREPO_CWD/minimal-repo"
+    "file://$OPAM_MONOREPO_CWD/repo"
+  ]
```

Comment:
I don't think printing it here makes sense. If you want to have the contents of the `pin-depends.opam` in the `run.t` I would suggest just creating it with `cat > pin-depends.opam <<EOF`, otherwise its just the same info duplicated.